usr/src/man/man8/altexec.8

```diff
@@ -1,4 +1,4 @@
-n.\"
+.\"
```

Comment:
OH FOR CRYING OUT LOUD, stupid "I didn't press 'n'o in emacs, right?" bug.  I'm sorry.